services/ui/ws/platform_display_default.cc

```diff
@@ -76,11 +76,15 @@ void PlatformDisplayDefault::Init(PlatformDisplayDelegate* delegate) {
   NOTREACHED() << "Unsupported platform";
 #endif
 
-  platform_window_->Show();
   if (image_cursors_) {
     image_cursors_->SetDisplay(delegate_->GetDisplay(),
                                metrics_.device_scale_factor);
   }
+  // Show() must be called only after |image_cursors_| has created a
+  // a cursor loader after display is sent to it. Otherwise, Show()
+  // triggers a delegate call to OnBoundsChanged(), which end up to
+  // changing cursor location for not existing cursor loader.
+  platform_window_->Show();
```

Comment:
does that fix occasional a crash at start up? if so, can it be sent as a separate PR?